spectrocloud/resource_registry_oci_ecr.go

```diff
@@ -46,6 +49,10 @@ func resourceRegistryOciEcr() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"provider_type": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
Can we make this as optional and set as "helm" to default as similar to UI. if we made it as required. if customer bump up the tf version they need to update their configurations

Reply:
Done.